### PR TITLE
Don't accept assignments from bots

### PR DESCRIPTION
**Pull request by dirk**

### CODE REVIEW

File: lib/lita/handlers/whoami.rb [hunk 1]
```diff
@@ -15,16 +16,14 @@ class Whoami < Handler
         "who is PERSON" => "Everbot will tell you who somebody is."
       })
 
-      route(/^(i|I) don['’‘]t know who anyone is/, :describe_everyone, command: true, help: {
-        "who is PERSON" => "Everbot will tell you who somebody is."
-      })
+      route(/^(i|I) don['’‘]t know who anyone is/, :describe_everyone, command: true)
```
Comment (taylorlapeyre) [inline lib/lita/handlers/whoami.rb]: Why not correct the help text instead of removing it?

File: lib/lita/handlers/whoami.rb [hunk 2]
```diff
@@ -62,6 +62,12 @@ def get_descriptiors_for(name)
         redis.lrange(key_for_person(name), 0, -1).uniq
       end
 
+      def valid_thing?(thing, chat)
+        return false if config.bots.include?(chat.message.source.user.name)
+
+        true
+      end
```
Comment (taylorlapeyre) [inline lib/lita/handlers/whoami.rb:69]: how about name this method `invalid_thing?`

``` ruby
config.bots.include?(chat.message.source.user.name)
```

**Comment by dirk** [comment]
@taylorlapeyre: Comments addressed.

File: lib/lita/handlers/whoami.rb [hunk 3]
```diff
@@ -62,6 +64,10 @@ def get_descriptiors_for(name)
         redis.lrange(key_for_person(name), 0, -1).uniq
       end
 
+      def invalid_thing?(thing, chat)
+        true if config.bots.include?(chat.message.source.user.name)
```
Comment (taylorlapeyre) [inline lib/lita/handlers/whoami.rb]: you don't need the `true if` :smile:

**Comment by dirk** [comment]
@taylorlapeyre: Bump. :wink:

**Pull request merged**